minhphan156/Abode
# SJSU-1047_1 review and rating button

**What I did:**
-implemented review and rating button
-connected the button with backend

**How to test:**
-log in and checkout a hotel with your email
-talk to Caffrey to change the booking status in the backend so the review button can be rendered
-leave star ratings and comments
-log out and log back in should show you ratings you left

**Files that need to review:**
ReviewRating.js
reviewActions.js
profileReducer.js 
HistoryOverview.js
